doc/development.md

```diff
@@ -3,7 +3,7 @@
 ## Technology stack required for development
 
 * [operator-sdk] version v1.28.1
```

Comment:
I know it is not related to this PR,

Can you update operator-sdk] version to v1.32.0?

Reply:
Yeah Sure